docs/history/5_x/version-5.6.0.md

```diff
@@ -21,6 +21,10 @@ Release Date: September 12, 2024
 
 <VidStack src="youtube/AVf0Pwafza0"/>
 
+::: caution
+This release contains a critical issue where scheduled jobs are no longer executed after making unrelated edits to the job.  This can happen when the jobs is edited via API, job/project imports, SCM, or direct job editing.  It is strongly recommended to use or upgrade to [version 5.6.1](version-5.6.1.md).
```

Comment:
```suggestion
This release fixes a critical issue where scheduled jobs are no longer executed after making unrelated edits to the job.  This can happen when the jobs is edited via API, job/project imports, SCM, or direct job editing.  It is strongly recommended to use or upgrade to [version 5.6.1](version-5.6.1.md).
```

Reply:
5.6.0 doesn't fix it.  5.6.1 does.

Reply:
oops, didn't notice the verion